scripts/prepare-environment-k3d.sh

```diff
@@ -29,13 +29,30 @@ function create_docker_pull_secret {
 	fi
 }
 
+function prepare_system_domain {
+  if [[ -z "${EPINIO_SYSTEM_DOMAIN}" ]]; then
```

Comment:
The dev documentation should contain this information as well ... Especially the quickstart guide, and for docs related to k3d/k3s.

Reply:
I was thinking that maybe I should have made viper/cobra aware of the `SYSTEM_DOMAIN` environment variable (like we do for other flags) and use that instead. I'm not sure we want that outside dev workflow. I will update the docs like you said.

Reply:
https://github.com/epinio/epinio/pull/983/commits/5dd87585b5bc3d14bd5cc0938008a0ca5e1db07d